packages/codegen/__tests__/codegen.test.ts

```diff
@@ -0,0 +1,91 @@
+import type { Project } from 'fixturify-project';
+import * as fs from 'node:fs/promises';
+import * as path from 'node:path';
+import { afterEach, beforeEach, describe, expect, test } from 'vitest';
+import { athenaCodegen } from '../src/index.js';
+import { createFixtures } from './utils/project.js';
+
+describe('it works', () => {
```

Comment:
would be great to add a test to illustrate hookable query hash generation

Reply:
i'll add one